tvtk/vtk_module.py

```diff
@@ -22,3 +22,12 @@
     from tvtk_local import *
 except ImportError:
     pass
+
+
+vtk_version = vtkVersion.GetVTKVersion()
+SKIP = []
+
+if vtk_version in ['9.0.3', '9.0.2']:
+    # Cause problems if used so ignore these.
+    SKIP = ['vtkDataEncoder', 'vtkWebApplication']
+    del vtkDataEncoder, vtkWebApplication
```

Comment:
What do you think about making this a dictionary, mapping vtk versions to the classes that should be skipped. That will be useful if we expect this to happen again in the future.

Reply:
Yes, we could but this sort of massive failure has not happened before, while wrapping has been a problem it hasn't been the case that just instantiating a class and calling some of its methods causes a lockup thus far.  So I am hoping this is not necessary going forward.  If it is, perhaps that would be a good time to redo this? Does that sound reasonable?  Right now I am seeing this as a one-off which hopefully does not recur.  We already have other issues to deal with with 9.1.x.